packages/portfolio-contract/src/type-guards.ts

```diff
@@ -150,6 +151,11 @@ export const PoolPlaces = {
   Compound_Polygon: { protocol: 'Compound', chainName: 'Polygon' },
   Compound_Fantom: { protocol: 'Compound', chainName: 'Fantom' },
   Compound_Binance: { protocol: 'Compound', chainName: 'Binance' },
+  Beefy_re7_Avalanche: {
```

Comment:
hm... or `Beefy_Avalanche_re7`? I don't think either is incorrect; but it sets a precedent. I'm not sure which way is better. I suppose it's OK as is.